src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
I guess VS did something weird here, but it seems like all the changes in this file should be reverted.